linera-service/src/project.rs

```diff
@@ -89,19 +91,20 @@ impl Project {
     }
 
     fn install_test_runner() -> Result<()> {
-        println!("installing test runner...");
+        info!("installing test runner...");
         let cargo_install = Command::new("cargo")
-            .args(["install", "linera-test-runner"])
+            .args(["install", "linera-sdk"])
+            .args(["--bin", RUNNER_BIN_NAME])
             .spawn()?
             .wait()?;
         if !cargo_install.success() {
-            bail!("failed to install linera-test-runner")
+            bail!("failed to install {}", &RUNNER_BIN_NAME)
```

Comment:
Would be useful to print `cargo`'s output in that case.

Reply:
Good point. The cargo stdout/stderr is piped by default so the user should see it.

Reply:
Doesn't piped just mean that our process receives it, and _can_ output it if needed, i.e. that it ends up in `output.stdout`?

Reply:
Hm :thinking: When it tries to `cargo install` I do see stdout / stderr. Let me double check.